lib/tilex/web/templates/feed/index.xml.eex

```diff
@@ -15,6 +15,9 @@
         <description>
           <![CDATA[<%= Tilex.Markdown.to_html(post.body) %>]]>
         </description>
+        <dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">
```

Comment:
Any thoughts on moving `xmlns:dc="http://purl.org/dc/elements/1.1/"` into the `<rss>` tag? so line 2 would be:

 ```
<rss version="2.0"
  xmlns:atom="http://www.w3.org/2005/Atom"
  xmlns:dc="http://purl.org/dc/elements/1.1/">
```